src/layout/_template.njk

```diff
@@ -44,6 +42,15 @@
     {% set page_title = "ONS Design System" %}
 {% endif %}
 
+{# Page columns #}
+{% set mainColNumber = pageConfig.mainCol.columns | default("8") %}
+{% set mainColClasses = (' ' + pageConfig.mainCol.colClasses) if pageConfig.mainCol.colClasses is defined and pageConfig.mainCol.colClasses %}
+{% set mainClasses = ' ' + pageConfig.mainCol.classes if pageConfig.mainCol.classes is defined and pageConfig.mainCol.classes %}
```

Comment:
Might need to put an if around this to check that `pageConfig.mainCol` exists here to avoid potential errors in jinja

Reply:
Will it need this when it has a default value? It doesn't _need_ to exist